### ISSUE TITLE
Some accessibility issues

### ISSUE THREAD

**Issue by zhou-zhoukang** [adrcotfas/goodtime#400]
Hi! Thanks for your hard work on this great open-source app. I noticed a few accessibility improvements that could be made.

1. Some ImageButton elements in the "Support development" screen are missing ContentDescriptions. Without content descriptions, users with visual impairments may have difficulty understanding the purpose of these buttons when using screen readers.

2. Adding an `onClickLabel` to the Button might also be a good practice. An example of how this can be implemented is provided in the code snippet below. For instance, when focusing a Button, a screen reader might announce "Double tap to **activate**" before a label is added. After adding an appropriate label, it would instead say "Double tap to **Buy me a coffee**", which can make the switch's function more intuitive and clear to users. 

```
modifiler = Modifier.semantics {  
   onClick(label = "Buy me a coffee") { }
}
```

Please don’t feel pressured in any way — I completely understand if other priorities take precedence. I’m truly grateful for all the effort and hard work you’ve already invested.
